android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java

```diff
@@ -345,6 +345,10 @@ public void openContactForm(ReadableMap contact, Callback callback) {
 
         if(thumbnailPath != null && !thumbnailPath.isEmpty()) {
             Bitmap photo = BitmapFactory.decodeFile(thumbnailPath);
+            if(photo == null) {
+                // Try to find the image in assets
+                photo = getBitmapFromAsset(thumbnailPath);
+            }
```

Comment:
These four lines should be extracted to a method. (They are reused 3 times)

Bitmap photo = BitmapFactory.decodeFile(thumbnailPath);
if(photo == null) {
    // Try to find the image in assets
    photo = getBitmapFromAsset(thumbnailPath);
}

But see the comment below. Is there a possibility to use asset-library url instead?

Reply:
Yes. Will put this in a separate method